includes/common.inc.php

```diff
@@ -135,6 +135,11 @@
     die('ERROR: Authentication failed ('.$server['host'].':'.$server['port'].')');
   }
 }
+if (isset($server['username']) && isset($server['password'])) {
+  if (!$redis->auth($server['username'],$server['password'])) {
+    die('ERROR: Authentication failed ('.$server['host'].':'.$server['port'].')');
```

Comment:
Should the return value of `auth` also be included as that contains a hint to what went wrong?